documentation/architecture.md

```diff
@@ -0,0 +1,247 @@
+# Clear Containers Architecture
+
+## Overview
+
+This is an architectural overview of Clear Containers, based on the 2.1 release.
+
+The [Clear Containers runtime (cc-oci-runtime)](https://github.com/01org/cc-oci-runtime)
+complies with the [OCI](https://github.com/opencontainers) specifications and thus
+work seamlessly with the [Docker Engine](https://www.docker.com/products/docker-engine)
+pluggable runtime architecture. In other words, one can transparently replace the
+[default Docker runtime (runc)](https://github.com/opencontainers/runc) with `cc-oci-runtime`.
+
+![Docker and Clear Containers](https://github.com/01org/cc-oci-runtime/blob/topic/documentation/documentation/docker-cc.png)
```

Comment:
oh, and it would be excellent if we could have the 'source' for the images commit as well, to help us make future edits etc.